src/App.tsx

```diff
@@ -1,11 +1,69 @@
-import { useState } from 'react';
+import { useCallback, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
+import { getMovie } from './api';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [query, setQuery] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isMovieFound, setIsMovieFound] = useState(false);
+
+  const resetForm = useCallback(() => {
```

Comment:
and also use consistent naming
```suggestion
  const handleFormReset = useCallback(() => {
```